### DISCUSSION TITLE
Youtube flickering problem

### DISCUSSION THREAD

**Discussion by jarno9981** [cefsharp/CefSharp#3355] [General]
When i use youtube in .net 4.7.2 framework with the pre-release cefsharp every 3 secondes cefsharp control vidoe wil flicker above 720p is there a fix for that using c#

**Comment by jarno9981** [comment]
cefsharp 87.1.130-pre

**Comment by amaitland** [comment]
It's unlikely a bug in CefSharp, more likely am upstream issue. In future 

Please stop creating issues. If this is a bug in CefSharp then the original issue will be reopened. 

How many machines have you tested with?
What version of Windows?
What graphics card?
Are you using WinForms or WPF? 

You can download and test with the CEF Sample application 


**Does this problem also occur in the `CEF` Sample Application**
    - Download one of the following:
    - For x86 download https://cef-builds.spotifycdn.com/cef_binary_87.1.13%2Bg481a82a%2Bchromium-87.0.4280.141_windows32_client.tar.bz2
    - For x64 download https://cef-builds.spotifycdn.com/cef_binary_87.1.13%2Bg481a82a%2Bchromium-87.0.4280.141_windows64_client.tar.bz2
	- Extract and run cefclient.exe
		- If you are using WPF/OffScreen run
		```
		cefclient.exe --multi-threaded-message-loop --off-screen-rendering-enabled --enable-gpu --no-sandbox --disable-site-isolation-trials
		```
		- If you are using WinForms run
		```
		cefclient.exe --multi-threaded-message-loop --no-sandbox --disable-site-isolation-trials
		```
	- **MAKE SURE TO TEST WITH THE COMMAND LINE ARGS LISTED ABOVE**
    - If you can reproduce the problem with `cefclient` then please report the issue on https://bitbucket.org/chromiumembedded/cef/overview (Make sure you search before opening an issue). If you open an issue here it will most likely be closed as `upstream` as the bug needs to be fixed in `CEF`.

**Reply by amaitland** [reply]
If cefclient runs successfully then please test with the minimal example. 

https://github.com/cefsharp/CefSharp.MinimalExample/tree/cefsharp/87 (making sure to use the cefsharp/87 branch).

**Comment by jarno9981** [comment]
X86 32bit
Windows 12791-1000 (rs_release)
Gtx 1660ti
Winforms c#


Op vr 15 jan. 2021 20:36 schreef Alex Maitland <notifications@github.com>:

**Reply by amaitland** [reply]
Download https://cef-builds.spotifycdn.com/cef_binary_87.1.13%2Bg481a82a%2Bchromium-87.0.4280.141_windows32_client.tar.bz2

Test with cefclient.exe --multi-threaded-message-loop --no-sandbox --disable-site-isolation-trials
